static/app/views/monitors/monitors.tsx

```diff
@@ -37,6 +38,10 @@ type State = AsyncView['state'] & {
 class Monitors extends AsyncView<Props, State> {
   getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
     const {params, location} = this.props;
+
+    trackAdvancedAnalyticsEvent('monitors.page_viewed', {
+      organization: params.orgId,
+    });
```

Comment:
Hmm I don't think it's great to add a side-effect to `getEndpoints` -- it'd be better to stick this in a lifecycle method

Reply:
Also this is going to break because `params.orgId` is a slug and not an actual ID, right?

Reply:
we have the full `organization`, let's use that

Reply:
ahh didn't know about the slug/id, but will change accordingly